core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/coprocessor/transactional/TrxRegionEndpoint.java.tmpl

```diff
@@ -4241,8 +4257,12 @@ CoprocessorService, Coprocessor {
                  e.add(kv);
                  try {
                     final WALKey wk = new WALKey(this.regionInfo.getEncodedNameAsBytes(), this.regionInfo.getTable(), EnvironmentEdgeManager.currentTime());
+#ifdef CDH5.7 APACHE1.2
+                    txid = this.tHLog.append(this.m_Region.getTableDesc(), this.regionInfo, wk, e, false);
```

Comment:
i do a test and find out there is no side effect
before cdh5.7
-----------------------------------------------------
2016-08-04 05:44:09,318 TRACE [B.defaultRpcServer.handler=17,queue=2,port=35946] transactional.TrxRegionEndpoint: TrxRegionEndpoint coprocessor: commit - txId 3, Write commit HLOG seq 274
----do sqstop && swsopall and do sqstart && swstartall
2016-08-04 06:44:48,680 TRACE [B.defaultRpcServer.handler=29,queue=2,port=43771] transactional.TrxRegionEndpoint: TrxRegionEndpoint coprocessor: commit - txId 3, Write commit HLOG seq 214

cdh 5.7
-------------------
2016-08-04 06:18:52,101 TRACE [PriorityRpcServer.handler=5,queue=1,port=58401] transactional.TrxRegionEndpoint: TrxRegionEndpoint coprocessor: commit - txId 3, Write commit HLOG seq 227